### PR TITLE
Reorg tutorials for managing rippled

### PR DESCRIPTION
**Pull request by mDuo13**
- Split off "enabling sharding" from the sharding concept.
- Divide the install and update instructions into separate pages
    - Also expand the install instructions with a little more context
    - Summarizes the recommended requirements from Capacity Planning in the System Requirements page
- Make subcategories under "Managing the rippled server" for:
    - Installation (including compiling and updating)
    - Configuration (including the newly split off history sharding tutorial and connecting your server to the test net tutorial)
    - Stand-Alone Mode
    - (Existing Troubleshooting category remains, but the SQLite fix is properly moved in as originally intended.)
- Add a new entry in the "Server Won't Start" troubleshooting page for the case when you switch shard store formats.

<s>Unfortunately this conflicts slightly with #469 and #470 so it may need to be rebased and have some of those changes manually re-applied.</s> Rebased to incorporate changes from those PRs.

### CODE REVIEW

File: content/tutorials/manage-the-rippled-server/installation/build-run-rippled-ubuntu.md [hunk 1]
```diff
@@ -9,16 +9,9 @@ Use these instructions to build a `rippled` executable from source version 1.1.0
 For information about building `rippled` for other platforms, see [Builds](https://github.com/ripple/rippled/tree/develop/Builds) in the `rippled` GitHub repository.
 
 
-## System Requirements
-
-**_To build `rippled`:_**
-
-You need a **minimum** of 8GB of RAM.
```
Comment (mDuo13) [inline content/tutorials/manage-the-rippled-server/installation/build-run-rippled-ubuntu.md:16]: The minimum requirements now include 8GB of RAM just to run rippled, so this separation is no longer relevant

File: content/_snippets/conf-file-location.md [hunk 2]
```diff
@@ -0,0 +1 @@
+The [recommended installation](install-rippled.html) uses the config file `/etc/opt/ripple/rippled.cfg` by default. Other places you can put a config file include `$HOME/.config/ripple/rippled.cfg` (where `$HOME` is the home directory of the user running `rippled`), `$HOME/.local/ripple/rippled.cfg`, or the current working directory from where you start `rippled`.
```
Comment (jhaaaa) [inline content/_snippets/conf-file-location.md:1]: THIS! Thank you for writing this up -- the Additional Configuration section in the post-rippled-install.md snippet needs this. That section currently lists the location of rippled.cfg for yum only -- but I think we should list this instead - all of the valid locations? Separate ticket and PR?

File: dactyl-config.yml [hunk 3]
```diff
@@ -876,92 +876,185 @@ pages:
...
-        html: start-a-new-genesis-ledger-in-stand-alone-mode.html
+    -   md: tutorials/manage-the-rippled-server/configuration/cluster-rippled-servers.md
+        html: cluster-rippled-servers.html
         funnel: Docs
         doc_type: Tutorials
         category: Manage the rippled Server
+        subcategory: Configuration
+        blurb: Set up a group of servers that share work for higher efficiency.
         targets:
             - local
```
Comment (jhaaaa) [inline dactyl-config.yml:1025]: I think we need to add the following?

>     -   md: tutorials/manage-the-rippled-server/configuration/configure-history-sharding.md
>         html: configure-history-sharding.html
>         funnel: Docs
>         doc_type: Tutorials
>         category: Manage the rippled Server
>         subcategory: Configuration
>         blurb: Set up a server to contribute to preserving historical XRP Ledger data without each server needing to store the full history a group of servers that share work for higher efficiency.
>         targets:
>             - local

**Comment by jhaaaa** [comment]
I really appreciate what this PR does. I think the new structure surfaces all of the correct keywords in the "Manage the rippled Server" left nav section. It makes it easier to find exactly what you need, as well as surfaces a truer picture of the content that we provide. I also like the way it focuses the content and encourages focus going forward.

**Review by jhaaaa — APPROVED** [review]
Left one comment about a file that I think needs to be added to the dactyl config. Other than that - this looks great! 🙌

File: content/_snippets/conf-file-location.md (hunk 2 above)
Comment (mDuo13) [reply content/_snippets/conf-file-location.md:1]: I can put it in this PR

File: content/tutorials/manage-the-rippled-server/installation/update-rippled-automatically-on-centos-rhel.md [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+
+These instructions assume you have already [installed `rippled` from the `yum` repository](install-rippled-on-centos-rhel-with-yum.html).
+
+To set up automatic updates, complete the following steps:
+
+1. Check that `/opt/ripple/bin/update-rippled.sh` exists. If it does not, [update manually](update-rippled-manually-on-centos-rhel.html).
+
+2. Install `crond`:
+
+        $ sudo yum install cronie
```
Comment (mellery451) [inline content/tutorials/manage-the-rippled-server/installation/update-rippled-automatically-on-centos-rhel.md:13]: is `cronie` a typo ?

**Review by mellery451 — APPROVED** [review]
left one comment - otherwise looks good.

File: content/tutorials/manage-the-rippled-server/installation/update-rippled-automatically-on-centos-rhel.md (hunk 4 above)
Comment (mDuo13) [reply content/tutorials/manage-the-rippled-server/installation/update-rippled-automatically-on-centos-rhel.md:13]: No, cronie is the name of a specific implementation of the cron daemon and also [the name of the package in the Red Hat repos](https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/7/html/system_administrators_guide/ch-automating_system_tasks#s1-Scheduling_a_Recurring_Job_Using_Cron).

File: content/tutorials/get-started/get-started-with-rippleapi-for-javascript.md [hunk 5]
```diff
@@ -117,7 +117,7 @@ In addition to RippleAPI-specific code, this script uses syntax and conventions
 const RippleAPI = require('ripple-lib').RippleAPI;
 ```
 
-The opening line enables [strict mode](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Strict_mode). This is purely optional, but it helps you avoid some common pitfalls of JavaScript. See also: [Restrictions on Code in Strict Mode](https://msdn.microsoft.com/library/br230269%28v=vs.94%29.aspx#Anchor_1).
+The opening line enables [strict mode](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Strict_mode). This is purely optional, but it helps you avoid some common pitfalls of JavaScript.
```
Comment (mDuo13) [inline content/tutorials/get-started/get-started-with-rippleapi-for-javascript.md:120]: Microsoft appears to have redirected their Strict Mode page to the Mozilla page already linked in this line, so I removed the Microsoft link.

File: content/_snippets/post-rippled-install.md [hunk 6]
```diff
@@ -0,0 +1,51 @@
+It can take several minutes for `rippled` to sync with the rest of the network, during which time it outputs warnings about missing ledgers.
+
+For information about `rippled` log messages, see [Understanding Log Messages](understanding-log-messages.html).
+
+After your `rippled` has synchronized with the rest of the network, you have a fully functional stock `rippled` server that you can use for local signing and API access to the XRP Ledger. Use [`rippled` server states](rippled-server-states.html) to tell whether your `rippled` server has synchronized with the network. You can use the [`rippled` commandline interface](get-started-with-the-rippled-api.html#commandline) to test this quickly:
+
```
Comment (jhaaaa) [inline content/_snippets/post-rippled-install.md:6]: Thank you for taking the time to do this! 🔥 And thank you dactyl for making this possible... Same for the refinement in Additional Configuration below.

File: content/tutorials/manage-the-rippled-server/installation/build-run-rippled-macos.md [hunk 7]
```diff
@@ -1,104 +1,10 @@
-# Install rippled
+# Build and Run rippled on macOS
```
Comment (jhaaaa) [inline content/tutorials/manage-the-rippled-server/installation/build-run-rippled-macos.md:1]: Yep - got it. 👍

File: content/tutorials/manage-the-rippled-server/configuration/connect-your-rippled-to-the-xrp-test-net.md [hunk 8]
```diff
@@ -0,0 +1,56 @@
...
+        # [validator_list_sites]
+        # https://vl.ripple.com
+        #
+        # [validator_list_keys]
+        # ED2677ABFFD1B33AC6FBC3062B71F1E8397C1505E1C42C64D11AD1B28FF73F4734
+
+3. Restart `rippled`.
+
+4. To verify that your `rippled` is connected to the XRP Test Net, use the [server_info method][] on your server and compare it to the results from a public server on the Test Net. The `seq` field of the `validated_ledger` object should be the same on both servers (possibly off by one or two, if it changed as you were checking).
+
```
Comment (jhaaaa) [inline content/tutorials/manage-the-rippled-server/configuration/connect-your-rippled-to-the-xrp-test-net.md:42]: So clear, so clean... Thank you!

**Comment by jhaaaa** [comment]
Thank you for taking on the extra task of incorporating the macOS and Test Net content into this reorg. Looks GREAT! 👍

**Pull request merged**